src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Runtime.Augments;
+using System;
+using System.Reflection;
+
+namespace System
+{
+    public partial class AppDomain
+    {
+        private static AppDomain domain = new AppDomain();
+        private AppDomain() {}
+        public static AppDomain CurrentDomain => domain;
+        public Assembly[] GetAssemblies() => AppDomainAugments.GetAssemblies();
+        public string BaseDirectory => "";
+        public string RelativeSearchPath => "";
```

Comment:
What these properties do in CoreCLR today is irrelevant. The code for them happens to be there as internal detail, not exposed as public surface. The implementation is not necessarily correct.

The behavior of these properties on other runtimes is what matters. On other runtimes, the BaseDirectory property returns the "application directory", not an empty string, even for simple console applications without AppDomainSetup and AppDomainManager  The "application directory" is already returned by AppContext.BaseDirectory, so the public AppDomain implementation should just forward to it.